gcc/testsuite/rust.test/xfail_compile/continue1.rs

```diff
@@ -1,11 +1,11 @@
-/* { dg-error "cannot `continue` outside of a loop" "" { target { *-*-* } } 8 } */
-/* { dg-error "failed to type resolve expression" "" { target { *-*-* } } 8 } */
 fn main() {
```

Comment:
Hmm, I still need `dg-excess-errors` in this file, to avoid the corresponding `FAIL`.

Reply:
That's what I would like to better understand. Sometime, simply moving dg-error directives raise the excess-error. I don't really understand exactly why it's needed for some tests and for some others not. It's strange that you need it, I'll rerun it to check that's not needed on my side..

Reply:
After re-running with this single commit, I have the same behavior. So, not really sure what caused some previous run to be ok without the `dg-excess-errors`.